debian/systemd.install

```diff
@@ -17,6 +17,7 @@ lib/udev/rules.d/71-seat.rules
 lib/udev/rules.d/99-systemd.rules
 usr/bin/systemd-cgls
 usr/bin/systemd-cgtop
+usr/bin/systemd-coredumpctl
```

Comment:
I think debian sets that to be `/usr/bin` because IIRC I added this line after running `debuild` and seeing it complain about the untracked files.